scripts/generate_style.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
```

Comment:
If you use `import` or `export`, then you syntactically know that it has to be a module and is therefore automatically in strict mode.

Reply:
Oh, we could remove a lot of `use strict` from this codebase then.

Reply:
I just blindly pepper it on the top of every .js file because at some point someone told me I should do that :-D

Reply:
I removed the "use strict" from the two files I added.